packages/perseus/src/widgets/numeric-input/numeric-input.tsx

```diff
@@ -243,7 +243,7 @@ export class NumericInput
         });
 
         return (
-            <div>
+            <div className="Im-just-a-sweet-lil-numeric-input">
```

Comment:
Oops, need to remove this sweet lil' guy

Reply:
I'll actually be adding this back temporarily for aiding QA testing, and I will remove it prior to deploy.